data-files/samples/weapons.Experiment.Any

```diff
@@ -0,0 +1,151 @@
+                firePeriod = 0.3;
+                damagePerSecond = 4;
+                autoFire = false;
+            }; 
+            
+        }, 
+        { 
+            description = "2 shots"; 
+            id = "2shots"; 
+            settingsVersion = 1; 
+            trials = ( 
+                { 
+                    count = 20; 
+                    ids = ( "static", "static", "moving"); 
+                } ); 
+            
+            weapon = { 
+                id = "2hit"; 
+                firePeriod = 0.3;
+                damagePerSecond = 2;
+                autoFire = false;
+            }; 
+            
+        }, 
+        { 
+            description = "3 shots"; 
+            id = "3shots"; 
+            settingsVersion = 1; 
+            trials = ( 
+                { 
+                    count = 20; 
+                    ids = ( "static", "static", "moving"); 
+                } ); 
+            
+            weapon = { 
+                id = "3hit"; 
+                firePeriod = 0.3;
+                damagePerSecond = 1.5;
+                autoFire = false;
+            }; 
+            
+        }, 
+        { 
+            description = "autofire"; 
+            id = "autofire"; 
+            settingsVersion = 1; 
+            trials = ( 
+                { 
+                    count = 20; 
+                    ids = ( "static", "static", "moving"); 
+                } ); 
+            
+            weapon = { 
+                id = "4hit"; 
+                firePeriod = 0.2;
+                damagePerSecond = 1.8;
+                autoFire = true;
+            }; 
+            
+        }, 
+        { 
+            description = "fastfire"; 
+            id = "fastfire"; 
+            settingsVersion = 1; 
+            trials = ( 
+                { 
+                    count = 20; 
+                    ids = ( "static", "static", "moving"); 
+                } ); 
+            
+            renderWeaponStatus = false;
+            sceneHitSoundVol = 0;
+            sceneHitSound = "";
+            weapon = { 
+                id = "fast"; 
+                fireSound = "sound/fpsci_noise_50ms.wav";
+                fireSoundLoop = true;
+                firePeriod = 0;
+                damagePerSecond = 1.1;
+                autoFire = true;
+                missDecalTimeoutS = 3;
+            }; 
+            
+        }, 
+        { 
+            description = "show weapon"; 
+            id = "showweapon"; 
+            settingsVersion = 1; 
+            trials = ( 
+                { 
+                    count = 20; 
+                    ids = ( "static", "static", "moving"); 
+                } ); 
+            
+            weapon = { 
+                id = "show"; 
+                firePeriod = 0.3;
+                damagePerSecond = 1;
+                autoFire = false;
+                renderModel = True;
+                modelSpec = ArticulatedModel::Specification{			        // Default model
```

Comment:
Doing the full model spec here is descriptive, but may be overwhelming. IIRC technically you just need `renderModel = True` to get something drawn.

Reply:
While true, I think this is a nicer sample since it gives people the hooks they'd need to replace the model if they want to. In any case, this is a supplement to the docs, and hopefully not a replacement.